src/components/profile/style/profile.less

```diff
@@ -1,3 +1,5 @@
+@import url("https://netdna.bootstrapcdn.com/font-awesome/4.4.0/css/font-awesome.min.css");
```

Comment:
No need of this we already load font-awesome in [app.less](https://github.com/geoadmin/mf-geoadmin3/blob/master/src/style/app.less#L2)